index.php

```diff
@@ -12,16 +12,17 @@
 				$series_vol = $row ['series_vol'];
 				$comics->seriesInfo($series_id);
 				$series_issue_count = $comics->series_issue_count;
-				if ($series_issue_count == 1) {
-					$series_issue_count = $series_issue_count . ' Issue';
-				} else {
-					$series_issue_count = $series_issue_count . ' Issues';
-				}
 				$series_cover = $comics->series_latest_cover;
 				if ($series_cover == NULL) {
 					$series_cover = 'assets/nocover.jpg';
 				}
-				$series_list .= '<li class="col-xs-6 col-sm-3 col-md-2"><a href="issues.php?series_id=' . $series_id . '" class="series-info"><img src="/' . $series_cover  . '" /><div class="series-title"><h3>' . $series_name . '</h3></div></a><small>' . $series_issue_count . '</small><div class="volume-number"><span class="count">Vol ' . $series_vol . '</span></div><a href="#" class="button add-button">[Add New]</a><a href="#" class="button edit-button">[Edit]</a></li>';
+				$series_list .= '<li class="col-xs-6 col-sm-3 col-md-2">';
+				$series_list .= '<a href="issues.php?series_id=' . $series_id . '" class="series-info"><div class="comic-image">';
+				$series_list .= '<img src="/' . $series_cover  . '" alt="' . $series_name .'" />';
+				$series_list .= '<div class="series-title"><h3>' . $series_name . '</h3></div>';
+				$series_list .= '</div></a>';
+				$series_list .= '<small>' . $series_issue_count . '</small>';
+				$series_list .= '<div class="volume-number"><span class="count">Vol ' . $series_vol . '</span></div><a href="#" class="button add-button">[Add New]</a><a href="#" class="button edit-button">[Edit]</a></li>';
```

Comment:
Broke this up into concat'd strings to make it easier to read. My eyes were crossing with all that one line. :smile: